packages/web/docs/src/components/error-boundary.ts

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import { Component } from 'react';
+
+export class ErrorBoundary extends Component<{
```

Comment:
It is a bit absurd that this ins't exported from React, but I guess people have different error tracking needs and not everybody just plugs console.error into some useful log sink.